chapters/sampling-randomization-power.tex

```diff
@@ -269,62 +614,70 @@ \subsection{Randomization}
 
 %-----------------------------------------------------------------------------------------------
 
-\section{Clustering and stratification}
+\section{Advanced topics in research design and randomization}
```

Comment:
this seems to fit better as a subsection in the previous rather than a stand-alone, it's very tied to the random processes section

Reply:
Can do. But then we need to make _Clustering_ and _Stratification_ paragraphs and not subsections.

I implemented in 36406ce2625586e681754298834c7c7647aa27b0 and I think it works.